### DISCUSSION TITLE
Capacitor voltage rating?

### DISCUSSION THREAD

**Discussion by kpfleming** [hrvach/deskhop#51] [General]
The most readily available/lowest cost parts in the 1206 package are rated for 250V maximum, which is of course well above the 3.3V used in this circuit. From what I understand that doesn't really matter, but I'm curious what people have been using to populate their boards.

Capacitors with lower maximum voltage ratings are also much smaller packages, which would make sense for any sort of bulk production but not for hand soldering :)

**Comment by hrvach** [comment]
Important thing is to not have caps that are rated below the operating voltage level (and it's a good idea to leave a bit of slack there). 1206 was chosen on purpose to make it easier to solder, albeit 0805 is easy to solder as well and targeting that for the next board revision. :)
